app/services/badges/award_thumbs_up.rb

```diff
@@ -0,0 +1,48 @@
+module Badges
+  class AwardThumbsUp
+    THUMBS_UP_BADGES = {
+      1 => "100 Thumbs Up Milestone",
```

Comment:
I think we should rely not on `slug`s, not titles.

Also, do we create badges manually in the admin? So, other forems don't have it and the badges are not awarded? If this is the case, we should check for badges first, and return if they don't exist to avoid running other code.

Reply:
@lightalloy 

Good idea about checking first - I'll add that in.

I originally went with slugs but found difficulty writing the test - I needed a way of creating the badges in the test with the correct slug, but I couldn't do that without using the title. The slug is generated from the title, so they are intrinsically linked.